### DISCUSSION TITLE
macOS error

### DISCUSSION THREAD

**Discussion by Fido-vel-Fidel** [mucommander/mucommander#1021] [General]
Hi,
I have an error after update to 1.3.0 version on my mac macOS 13.4.1. After run I have this error "muCommander damaged and cannot be opened". I tried this solution:

_Note: as the application is not signed, the following error may appear when trying to start is on macOS: "muCommander damaged and cannot be opened".
This can be solved by executing: sudo xattr -r -d com.apple.quarantine /Applications/muCommander.app_

But problem still exist

**Comment by ahadas** [comment]
see #999
